cove_oc4ids/templates/cove_oc4ids/additional_checks_table.html

```diff
@@ -10,26 +10,71 @@
       </tr>
     </thead>
     <tbody>
-      {% for type, values in additional_checks.items %}
-        {% if type == 'empty_field' %}
+      {% for check in additional_checks %}
+        {% if check.check_id == 'missing-values' %}
           <tr>
             <td>{% trans 'The data includes fields that are empty or contain only whitespaces. Fields that are not being used, or that have no value, should be excluded in their entirety (key and value) from the data' %}</td>
             <td>
               <ul class="list-unstyled">
-                {% for value in values|slice:":3" %}
-                  <li>{{ value.json_location }}</li>
+                {% for path in check.paths|slice:":3" %}
+                  <li>{{ path }}</li>
                 {% endfor %}
-                {% if values|length > 3 %}
-                  <li><a data-toggle="modal" data-target=".{{"additional-checks-"|concat:forloop.counter}}"><sup>see all</sup></a></li>
+                {% if check.paths|length > 3 %}
+                  <li><a data-toggle="modal" data-target=".additional-checks-{{check.check_id}}"><sup>{% trans "see all" %}</sup></a></li>
                 {% endif %}
               </ul>
             </td>
           </tr>
         {% endif %}
+
+        {% if check.check_id == 'missing-currency' %}
+          <tr>
+            <td>
+              {% blocktrans count counter=check.paths|length %}
+              There is a value without a currency.
+              {% plural %}
+              There are {{ counter }} values without a currency.
+              {% endblocktrans %}
+
+              {% trans "Currencies should be published for all values." %}
+            <td>
+              <ul class="list-unstyled">
+                {% for path in check.paths|slice:":3" %}
+                  <li>{{ path }}</li>
+                {% endfor %}
+                {% if check.paths|length > 3 %}
+                  <li><a data-toggle="modal" data-target=".additional-checks-{{check.check_id}}"><sup>{% trans "see all" %}</sup></a></li>
+                {% endif %}
+              </ul>
+            </td>
+          </tr>
+        {% endif %}
+
+
+        {% if check.check_id == 'invalid-project-ids' %}
+          <tr>
+            <td>
+              {% blocktrans with count=check.paths|length %}
+              {{ count }} of your project <code>id</code> fields has a problem: There is no prefix or the prefix is not recognised
```

Comment:
We should probably say 

"There is no prefix or the prefix format is not recognised"

Since we don't recognise the _prefix_ itself (and to be consistent with OCDS).